exchanges/zb/zb_websocket.go

```diff
@@ -214,37 +215,42 @@ func (z *ZB) wsHandleData(respRaw []byte) error {
 			Status:   order.Cancelled,
 		}
 	case strings.Contains(result.Channel, "trades"):
-		var trades WsTrades
-		err := json.Unmarshal(fixedJSON, &trades)
+		if !z.IsSaveTradeDataEnabled() {
+			return nil
+		}
+		var tradeData WsTrades
+		err := json.Unmarshal(fixedJSON, &tradeData)
 		if err != nil {
 			return err
 		}
-
-		for i := range trades.Data {
-			channelInfo := strings.Split(result.Channel, "_")
+		var trades []trade.Data
+		for i := range tradeData.Data {
+			channelInfo := strings.Split(result.Channel, currency.UnderscoreDelimiter)
 			cPair, err := currency.NewPairFromString(channelInfo[0])
 			if err != nil {
 				return err
 			}
-
-			tSide, err := order.StringToOrderSide(trades.Data[i].TradeType)
+			var tSide order.Side
+			tSide, err = order.StringToOrderSide(tradeData.Data[i].TradeType)
```

Comment:
They have 2 fields, tradeData.Data[i].Type and tradeData.Data[i].TradeType

Type has buy/sell (which relates to order side in GCT) however their tradetype returns bid/ask which makes no sense.
Pls double check n if everything is as intended then just ignore this comment: https://www.zb.com/api#wugiixorsxsjshm60

Reply:
While the code does convert bid/ask to buy/sell now, I've updated this to use `Type` instead